spec/requests/conversion_hosts_spec.rb

```diff
@@ -76,44 +82,70 @@
       post(api_conversion_hosts_url, :params => sample_conversion_host_from_vm)
 
       expect(response).to have_http_status(:ok)
-      expect_result_resources_to_include_keys("results", expected_attributes)
 
-      conversion_host_id = response.parsed_body["results"].first["id"]
+      results = response.parsed_body["results"].first
 
-      expect(ConversionHost.find_by(:resource_id => vm.id).id).to eql(conversion_host_id.to_i)
+      expect(results['success']).to be_truthy
```

Comment:
This could be handled by a single expectation with `have_attributes`

Reply:
I was getting confused by all the different helper methods, which all behave slightly differently, so I just went with what I know.

Reply:
I can switch it to `include` but not `have_attributes` since the latter is looking for actual attributes i.e. methods, not keys. Anyway, updated.